src/views/Item/ItemWrapper.vue

```diff
@@ -5,9 +5,9 @@
         v-for="(tab, index) in tabs"
         :key="index"
         :to="{ name: tab.route }"
-        tag="button"
-        class="tabs__tab"
+        class="btn btn--sec tabs__tab"
         role="tab"
+        tabindex="0"
```

Comment:
Dette er vel default for `<a>`?

Reply:
Har du testet med annet enn Gecko? 🤔

Reply:
Mulig det er en Mac-greie faktisk 🤷

Reply:
Niks, testet bare med Firefox, men så det sto dette på MDN:

> Some focusable HTML elements have a default tabindex value of 0 set under the hood by the user agent. These elements are an `<a>` or [...]. Developers shouldn't add the tabindex attribute to these elements unless it changes the default behavior (for example, including a negative value will remove the element from the focus navigation order).

Men funker det ikke i WebKit uten å sette den?

Reply:
Så vidt jeg kan se er det visst en [MacOS-greie](https://www.a11yproject.com/posts/macos-browser-keyboard-navigation/). I tillegg til å være en system-wide "feature" virker det som de fire store™️ respekterer det på ulikt vis og har egne innstillinger for det (som ser ut til å være default av, kanskje med unntak av Chrome). Sikkert kjent for alle Mac-folk, og skal vel være sånn da, om jeg ikke misforstår helt 🤷